### PR TITLE
Fix the Date and Timestamp import issue (#45).

### PR DESCRIPTION
**Pull request by morazow**

### CODE REVIEW

File: hadoop-etl-common/src/test/java/com/exasol/hadoop/HdfsSerDeImportServiceDateTimeTest.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        }
+        rcWriter.writeValues(temporaryFile, values);
+
+        ExaIterator ctx = mock(ExaIterator.class);
+        final List<HCatTableColumn> columns = Arrays.asList(
+                new HCatTableColumn(type.getColumnName(), type.getColumnType()));
+        final List<OutputColumnSpec> outputColumns = OutputColumnSpecUtil
+            .generateDefaultOutputSpecification(columns, Collections.emptyList());
+
+        runImportRCFile(ctx, columns, Collections.emptyList(), outputColumns,
```
Comment (AnastasiiaSergienko) [inline hadoop-etl-common/src/test/java/com/exasol/hadoop/HdfsSerDeImportServiceDateTimeTest.java:117]: These two methods are very similar. Any chances to extract a common part to a method?

File: hadoop-etl-common/src/test/java/com/exasol/hadoop/RCFileWriter.java [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+import org.apache.hadoop.hive.ql.exec.FileSinkOperator.RecordWriter;
+import org.apache.hadoop.hive.ql.io.RCFileInputFormat;
+import org.apache.hadoop.hive.ql.io.RCFileOutputFormat;
+import org.apache.hadoop.hive.serde2.SerDeException;
+import org.apache.hadoop.hive.serde2.Serializer;
+import org.apache.hadoop.hive.serde2.columnar.ColumnarSerDe;
+import org.apache.hadoop.hive.serde2.objectinspector.ObjectInspector;
+import org.apache.hadoop.hive.serde2.objectinspector.ObjectInspectorFactory;
+import org.apache.hadoop.hive.serde2.objectinspector.SettableStructObjectInspector;
+import org.apache.hadoop.hive.serde2.objectinspector.StructField;
```
[the remark this answers is not in the document]
Comment (morazow) [reply hadoop-etl-common/src/test/java/com/exasol/hadoop/RCFileWriter.java:24]: Yes, but not regularly. I have added star import if more than 3 from same package.

File: hadoop-etl-common/src/main/java/com/exasol/hadoop/HdfsSerDeImportService.java [hunk 3]
```diff
@@ -551,10 +559,10 @@ private static Object getJavaObjectFromHiveString(String value, String type) thr
...
         case "java.sql.Timestamp":
-            obj = Timestamp.valueOf(value);
+            obj = java.sql.Timestamp.valueOf(value);
             break;
         case "java.sql.Date":
-            obj = Date.valueOf(value);
+            obj = java.sql.Date.valueOf(value);
             break;
         case "java.lang.Boolean":
             obj = Boolean.valueOf(value);
```
[the remark this answers is not in the document]
Comment (morazow) [reply hadoop-etl-common/src/main/java/com/exasol/hadoop/HdfsSerDeImportService.java:568]: Good idea! Fixed.

File: hadoop-etl-common/src/main/java/com/exasol/hadoop/HdfsSerDeImportService.java [hunk 4]
```diff
@@ -330,6 +330,14 @@ private static Object getJavaObjectFromPrimitiveData(Object data, ObjectInspecto
         } else if (obj instanceof byte[]) {
             obj = Hex.encodeHexString((byte[]) obj);
         }
+        if (obj instanceof Date) {
```
[the remark this answers is not in the document]
Comment (morazow) [reply hadoop-etl-common/src/main/java/com/exasol/hadoop/HdfsSerDeImportService.java:333]: Fixed. Combined them with if-else blocks.

File: hadoop-etl-common/src/test/java/com/exasol/hadoop/HdfsSerDeImportServiceDateTimeTest.java (hunk 1 above)
Comment (morazow) [reply hadoop-etl-common/src/test/java/com/exasol/hadoop/HdfsSerDeImportServiceDateTimeTest.java:108]: Extracted common parts.

**Pull request merged**